docs/integrations/pytorch_lightning.rst

```diff
@@ -1,318 +1,305 @@
-Neptune-PyTorch Lightning Integration
-=====================================
+PyTorch Lightning Integration
+=============================
 
-The integration enables you to log |PyTorch Lightning| metrics to Neptune.
+What will you get?
+------------------
+[video-placeholder]
 
-.. image:: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :target: ../_static/images/integrations/pytorchlightning_neptuneml.png
-   :alt: PyTorchLightning neptune.ai integration
+PyTorch Lightning is a lightweight PyTorch wrapper for high-performance AI research. With Neptune integration you can:
 
-Requirements
-------------
-This integration is enabled as part of PyTorch Ligthning loggers module, so all you need is to have |neptune-client| installed.
+* preview running experiment
+* log training and testing metrics, and visualize them in Neptune UI,
+* log experiment parameters,
+* monitor hardware usage,
+* log any additional metrics of your choice,
+* log performance charts,
+* save model checkpoints.
 
-.. code-block:: bash
+.. tip::
 
-    pip install neptune-client
+    Check this public project with example experiments: |project|.
 
+    You can also go straight to the |exp|.
 
-Create the **LightningModule**
-------------------------------
+Quickstart
+----------
 
-Nothing changes here, just your typical ``LightningModule`` structure.
+Introduction
+^^^^^^^^^^^^
+This quickstart will show you how to use Neptune with your existing PyTorch Lightning code. You have three options:
 
-.. code-block:: python3
+1. Follow through this quickstart for detailed explanations,
+2. Open starter code in Colab Notebook (badge-link below) and run it as a "`neptuner`" user - zero setup, it just works.
+3. View starter code as Python script on |script|.
 
-    class CoolSystem(pl.LightningModule):
+|Run on Colab|
 
-        def __init__(self):
-            ...
+Before you start
+^^^^^^^^^^^^^^^^
 
-        def forward(self, x):
-            ...
+**Prerequisites**
 
-        def training_step(self, batch, batch_idx):
-            ...
+* Python 3,
+* Neptune account |register|,
+* Minimal familiarity with the PyTorch Lightning.
 
-        def configure_optimizers(self):
-            ...
+**Supported version**
 
-        @pl.data_loader
-        def train_dataloader(self):
-            ...
+* ``pytorch-lightning>=0.9.0``
```

Comment:
I think we should have supported versions somewhere at the top, just below the ::tip:: perhaps?

Reply:
thx, corrected.